thavalon-server/thavalon_key.key

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
What if we don't checkin the private key?

Reply:
Eh. It's a self-signed certificate that no one in their right mind would use. Removed though, as you're ***technically*** correct.